workshops/build-a-binderhub/before_workshop.md

```diff
@@ -1,2 +1,26 @@
-A place to store the email sent out to attendees before the workshop.
-See #222 for a draft of the content.
+Hi!
+
+We are looking forward to welcoming you to the 'Build a Binderhub' workshop in Sheffield next week! We have a couple of things for you to look at before the workshop and some more information about what is happening on the day:
+
+## Getting ready
+We have funds available for travel and childcare support. Please get in touch with theturingway@gmail.com to discuss how we can support you.
+
+Before you arrive please make sure you have the following:
+* A “Free Trial” subscription on Microsoft Azure or an existing Azure account with an active subscription you would like to use
+* A [DockerHub account](https://hub.docker.com/) (free sign up)
+
+If you have time beforehand you may want to look at the installation requirements [here](https://github.com/alan-turing-institute/the-turing-way/blob/SG/binderhub-workshop/workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md#installation-requirements-) so that you can install the tools needed beforehand. This is not essential but it will save time on the day! 
+
+## On the day
+There is now an updated agenda: https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/boost-research-reproducibility-binder/agenda.md available and please note that we will begin registration at 9.30am and will formally close at 4pm so that those with caring responsibilities can get home!
+
+We are dedicated to an inclusive and supportive event. Please read the code of conduct before you come along: https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE_OF_CONDUCT.md. We encourage you to tweet and share your thoughts using #TuringWay, if you would not like to be included in any photographs we will have stickers you can use on your name badge to let us know!
+
+## Travelling to the University of Sheffield
+***Need to write this***
```

Comment:
Tagging @r-j-arnold to help...

Reply:
I've made a start on this, @r-j-arnold if you could add what you do when you get into Regent Court that would be great (I've never been in the CS bit!)

Reply:
Not pretty but it's done.

Reply:
Great, thank you!